core/pom.xml

```diff
@@ -156,7 +156,12 @@
             <classifier>models</classifier>
             <scope>provided</scope>
         </dependency>
-
+        <!-- Clustering attempt-->
+        <dependency>
+            <groupId>org.nield</groupId>
+            <artifactId>kotlin-statistics</artifactId>
+            <version>1.2.1</version>
+        </dependency>
```

Comment:
as said, version number should not be here. See `for_developers.md` for more details

Reply:
Minor oversight. Turns out I didn't need any of them, since I reimplemented the DBSCAN clusterer in kotlin. Removed the other attempt, but missed this one. Sorry.